easybuild/easyconfigs/v/vg/vg-1.50.1-foss-2022a.eb

```diff
@@ -0,0 +1,86 @@
+easyblock = 'MakeCp'
+
+name = 'vg'
+version = '1.50.1'
+
+homepage = 'https://github.com/vgteam/vg'
+description = """variation graph data structures, interchange formats,
+alignment, genotyping, and variant calling methods"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/vgteam',
+        'repo_name': '%(namelower)s',
+        'tag': 'v%(version)s',
+        'recursive': True,
+        'recurse_submodules': [
+            '!vcflib',
+            '!deps/htslib',
+            '!deps/tabixpp',
+        ],
+        'keep_git_dir': True,
+        'extra_config_params': [
+            'submodule."deps/elfutils".url=https://sourceware.org/git/elfutils.git',
+        ]
+    }
+}]
+patches = ['%(name)s-%(version)s_dep_fix.patch']
+checksums = [
+    None,
+    '4db8da4c7b4ed3c846f03ddab47a5b99345edc7db398882b12bcebc4a0ee3fc0',  # vg-1.50.1_dep_fix.patch
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('pybind11', '2.9.2'),
+    ('zstd', '1.5.2'),
+    ('Boost', '1.79.0'),
+    ('cairo', '1.17.4'),
+    ('lz4', '1.9.3'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.8.2'),
+    ('RDFlib', '6.2.0'),
+    ('UnZip', '6.0'),
+    ('cURL', '7.83.0'),
+    ('parallel', '20220722'),
+    ('bc', '1.07.1'),
+    ('jq', '1.6'),
+    ('libtool', '2.4.7'),
+    ('gettext', '0.21'),
+    ('Automake', '1.16.5'),
+    ('Jansson', '2.14'),
+    ('protobuf', '3.19.4'),
+    ('protobuf-python', '3.19.4'),
+    ('bzip2', '1.0.8'),
+    ('ncurses', '6.3'),
+    ('pkg-config', '0.29.2'),
+    ('XZ', '5.2.5'),
+    ('nodejs', '16.15.1'),
+
+    ('vcflib', '1.0.9', '-R-4.2.1'),
+    ('HTSlib', '1.15.1'),
+    ('tabixpp', '1.1.2'),
+]
+
+prebuildopts = 'mkdir lib && '
+prebuildopts += 'cp $EBROOTVCFLIB/lib/libvcflib.a lib/ && '
```

Comment:
I would add a comment here to explain why we're copying, and why that's OK:

```suggestion
# copy static libraries provided by dependencies to lib/, to avoid that they get built;
# note that symlinking does not work, because 'make' then may ignore the static libraries
# being found based on timestamps of sources vs those static libraries,
# and try to overwrite a static library of a dependency!
prebuildopts += 'cp $EBROOTVCFLIB/lib/libvcflib.a lib/ && '